src/System Application/App/Permission Sets/src/xmlports/ImportPermissionSets.xmlport.al

```diff
@@ -211,9 +222,20 @@ xmlport 9864 "Import Permission Sets"
                 }
 
                 trigger OnBeforeInsertRecord()
+                var
+                    MetadataPermission: Record "Metadata Permission";
+                    MetadataPermissionSetRel: Record "Metadata Permission Set Rel.";
                 begin
                     if TempMetadataPermissionSet.Get(TempMetadataPermissionSet."App ID", TempMetadataPermissionSet."Role ID") then
                         currXMLport.Skip();
+                    if not UpdatePermissions then begin
+                        MetadataPermissionSetRel.SetFilter("App ID", TempMetadataPermissionSet."App ID");
+                        MetadataPermissionSetRel.SetFilter("Role ID", TempMetadataPermissionSet."Role ID");
+                        MetadataPermissionSetRel.DeleteAll;
+                        MetadataPermission.SetFilter("App ID", TempMetadataPermissionSet."App ID");
+                        MetadataPermission.SetFilter("Role ID", TempTenantPermissionSet."Role ID");
+                        MetadataPermission.DeleteAll;
```

Comment:
The CodeCop is requesting paranthesis after all of the new `DeleteAll;` statements.